public/assets/css/main.css

```diff
@@ -78,6 +78,31 @@ figcaption {
 	box-shadow: rgb(157, 18, 60) 0 0 .2em;
 }
 
+ul {
+	list-style-type: none;
+	font-size: 80%;
+}
+
+ul li {
```

Comment:
Why not put this in footer li ?

Reply:
Same reason for `li` and `ul`, multiple small pull requests but similar css (as described above).